@re-/model/demos/generateCode/helpers.ts

```diff
@@ -0,0 +1,28 @@
+import { CommentRange } from "ts-morph"
+import { regex } from "./regex.js"
+
+export const addBlockToMaps = (
+    destinations: string[],
+    blockName: string,
+    text: string,
+    mappedData: Record<string, object>
+) => {
+    let dataToAdd: Record<string, string> = {}
```

Comment:
`let` should be `const` here since we're not reassinging to this variable.

This also made me realize that eslint is disabled for all of this code since it is in a folder called "demos" even though it is actually source code that is generating demos.

I mentioned in another comment moving this tool to `@re-/node`. Once you do that, you might have a couple other linter errors to fix. Sorry 😿